File: server/HeroesAPI/HeroesAPI/Models/Hero.cs
using Newtonsoft.Json;

namespace HeroesAPI.Models
{
    public class Hero
    {
        [JsonProperty(PropertyName = "id")]
        public Guid Id { get; set; }
        public string Name { get; set; }
        public Power Power { get; set; }
        public string? AlterEgo { get; set; }
    }
}

File: client/src/app/interfaces/hero.ts
import { Power } from './power';

export interface Hero {
  id: string;
  name: string;
  power: Power;
  image?: string;
  alterEgo?: string;
}

export interface NewHeroForm {
  name: string;
  power: Power;
  image?: string;
  alterEgo?: string;
}
